eth/tracers/api.go

```diff
@@ -1105,6 +1105,10 @@ func overrideConfig(original *params.ChainConfig, override *params.ChainConfig)
 		copy.EtnaTimestamp = timestamp
 		canon = false
 	}
+	if timestamp := override.FUpgradeTimestamp; timestamp != nil {
+		copy.FUpgradeTimestamp = timestamp
+		canon = false
+	}
```

Comment:
Should this be after the Cancun check (because Cancun was ~mostly activated with Etna)

Reply:
Let's keep our upgrades separate from the eth ones.